frontend/package.json

```diff
@@ -15,6 +15,7 @@
     "gulp-protractor": "0.0.11",
     "gulp-replace": "^0.5.3",
     "gulp-ruby-sass": "^0.7.1",
+    "gulp-util": "^3.0.4",
```

Comment:
@hirtie-maxim what's this for? if this is a fix for something else, it should be part of another PR.